syil_lnc_toolpath.cps

```diff
@@ -1673,23 +1675,49 @@ function onCyclePoint(x, y, z) {
       writeBlock(gFormat.format(65), '"PROTECTEDMOVE"', zOutput.format(z - cycle.depth));
 
       WCS_CODE = getProbingArguments(cycle, probeWorkOffsetCode);
+    
+      if(cycle.approach1== "positive")
```

Comment:
I think you could replace the `if` statement here with the `approach` function. That converts it to a sign, which you could use to have only a single set of equations.